cfme/tests/test_appliance.py

```diff
@@ -99,7 +99,7 @@ def test_service_enabled(appliance, service):
         casecomponent: Appliance
     """
     if service == 'postgresql':
-        service = '{}-postgresql'.format(appliance.db.postgres_version)
+        service = str(appliance.db_service.unit_name)
```

Comment:
I don't think this is a required change. There's no harm in the call to `str()` here, and the test is being explicit in what its expecting `service` to be - a string, not some object type. Its possible unit_name is not actually a string type.

Reply:
@mshriver accept is not required comment :)
I don't understand why extra call? I remember one PR to avoid the extra call and what should be other probable `type` of `unit_name`? I'm not able to understand.